src/client/ClientDataResolver.js

```diff
@@ -84,7 +84,7 @@ class ClientDataResolver {
    * Data that resolves to give a GuildMember object. This can be:
    * * A GuildMember object
    * * A User object
-   * @typedef {Guild} GuildMemberResolvable
+   * @typedef {GuildMember} GuildMemberResolvable
```

Comment:
The point of the resolver (which I did not write) is to resolve users to GuildMembers. GuildMembers are compatible just for code simplification...